src/content/developers/docs/networks/index.md

```diff
@@ -58,6 +58,7 @@ The Sepolia network uses a permissioned validator set. It's fairly new, meaning
 - [PoW faucet](https://sepolia-faucet.pk910.de/)
 - [Sepolia faucet](https://faucet.sepolia.dev/)
 - [FaucETH](https://fauceth.komputing.org)
+- [Coinbase Wallet Faucet | Sepolia](https://coinbase.com/faucets/ethereum-sepolia-faucet)
```

Comment:
```suggestion
- [Coinbase Wallet Faucet](https://coinbase.com/faucets/ethereum-sepolia-faucet)
```

Just going to remove the network portion in this, should be pretty self-explanatory given the context of the copy.